tutorials-in-development/Staging_Area/01-explore-phenology-data.Rmd

```diff
@@ -614,7 +617,7 @@ phenoPlot
 
 We could also covert this to percentage and plot that. 
 
-```{r plot-leaves-percentage}
+```{r plot-leaves-percentage, fig.cap="Bar plot showing the proportion of Liriodendrum tulipifera (LITU) individuals from January 2017 through December 2019 at the Smithsonian Conservation Biology Institute (SCBI). The y-axis represents the percent of individuals that were recorded as a "yes" for the phenophase of interest,"Leaves", and were from the primary plots."}
```

Comment:
Many of these figure captions are very similar - though it looks like there are slight differences in the plots (that will be hard to detect for people with limited vision). Could you please add a sentence to each that describes the subtle differences between the plots? You do this well above with: " Axis labels and title have been added to make the graph more presentable."